src/fn/upload-custom-translations.js

```diff
@@ -81,6 +89,34 @@ function parse(filePath, options) {
   });
 }
 
+function checkTranslations(translations, languageCode) {
+  let mf = null;
+  try {
+    mf = new MessageFormat(languageCode);
+  } catch (e) {
+    log.warn(`Cannot check '${languageCode}' translations: ${e.message}`);
+  }
+  let errorsFound = 0;
+  for (const [msgKey, msgSrc] of Object.entries(translations)) {
+    if (!msgSrc) {
+      log.warn(`Empty '${languageCode}' translation for '${msgKey}' key`);
```

Comment:
I just realised this will spam the console for projects that have an empty language (eg: one they're still working on). Maybe keep a count of `emptyTranslations` and do one warning at the end with `${emptyTranslations} empty translations found for ${languageCode}`?

Reply:
Yes, taking into account that it is easy to find yourself the empty ones with an editor, it is a good idea.